service/controller.go

```diff
@@ -2132,11 +2139,83 @@ func (s *service) GetCapacity(
 			"Unable to get capacity: %s", err.Error())
 	}
 
+	Log.Println("defaulttttttttt", s.opts.defaultSystemID)
```

Comment:
Typo * => Log.Println("defaulttttttttt", s.opts.defaultSystemID), Please fix this.

Reply:
Done